app/src/main/java/com/yeonae/chamelezone/LoginActivity.kt

```diff
@@ -0,0 +1,69 @@
+package com.yeonae.chamelezone
+
+import android.content.DialogInterface
+import android.content.Intent
+import android.os.Bundle
+import android.widget.Toast
+import androidx.appcompat.app.AlertDialog
+import androidx.appcompat.app.AppCompatActivity
+import kotlinx.android.synthetic.main.activity_login.*
+
+class LoginActivity : AppCompatActivity() {
+    val testEmail = "heimish_08@naver.com"
+    val testPassword = "1234"
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_login)
+        btn_find_email.setOnClickListener {
+            val intent = Intent(this, FindEmailActivity::class.java)
+            startActivity(intent)
+        }
+        btn_find_password.setOnClickListener {
+            val intent = Intent(this, FindPasswordActivity::class.java)
+            startActivity(intent)
+        }
+        btn_join.setOnClickListener {
+            val intent = Intent(this, JoinActivity::class.java)
+            startActivity(intent)
+        }
+        btn_login.setOnClickListener {
+            loginCheck("${edt_email.text}", "${edt_password.text}")
+        }
+    }
+
+    fun loginCheck(email: String, password: String) {
+        if(email.isEmpty()){
+            Toast.makeText(applicationContext, "아이디를 입력해주세요!", Toast.LENGTH_LONG).show()
+        }
+        else if(password.isEmpty()){
+            Toast.makeText(applicationContext, "비밀번호를 입력해주세요!", Toast.LENGTH_LONG).show()
+        }
+        else{
+            if (email == testEmail) {
+                if (password == testPassword) {
+                    Toast.makeText(applicationContext, "로그인 성공", Toast.LENGTH_LONG).show()
+                } else {
+                    alertdialog()
+                }
+            } else {
+                alertdialog()
+            }
+        }
+    }
+
+    fun alertdialog() {
```

Comment:
이 함수 여러페이지에서 사용되는데 
```suggestion
    fun alertdialog(message: String) {
```
이런식으로 message만 받아서 재사용하도록 class를 분리해보면 좋을거같아요

Reply:
DialogFragment로 수정했습니다